e2e/connection_test.go

```diff
@@ -181,6 +200,27 @@ func consumerConnectFlow(t *testing.T, tequilapi *tequilapi_client.Client, consu
 
 	// call the custom asserter for the given service type
 	serviceTypeAssertionMap[serviceType](t, se)
+
+	consumerStatus, err := tequilapi.IdentityStatus(consumerID)
+	assert.NoError(t, err)
+	assert.True(t, consumerStatus.Balance < uint64(690000000), "balance should decrease but is ", consumerStatus.Balance)
+	assert.Equal(t, uint64(0), consumerStatus.Earnings)
+	assert.Equal(t, uint64(0), consumerStatus.EarningsTotal)
+
+	return uint64(690000000) - consumerStatus.Balance
+}
+
+func providerEarnedTokens(t *testing.T, tequilapi *tequilapi_client.Client, id string, earningsExpected uint64) {
+	// Before settlement
+	providerStatus, err := tequilapi.IdentityStatus(id)
+	assert.NoError(t, err)
+	assert.Equal(t, uint64(690000000), providerStatus.Balance)
+	assert.Equal(t, earningsExpected, providerStatus.Earnings)
+	assert.Equal(t, earningsExpected, providerStatus.EarningsTotal)
+	// TODO Compare with sessions stats + proposal price
+	assert.True(t, providerStatus.Earnings > uint64(500), "earnings should be at least 500 but is ", providerStatus.Earnings)
```

Comment:
If you're providing args, this should be a format string, hence `%d` is missing. As it is now, it will print: 
```
earnings should be at least 500 but is %!(EXTRA int=5)
```

Reply:
Added